### ISSUE TITLE
Button: Larger touch target

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystem#537]
Touch targets are the parts of the screen that respond to user input. They extend beyond the visual bounds of an element. For example, a button may appear to be 48 x 36 px, but the padding surrounding it comprises the full 48 x 48 px touch target.

We should update our button component with a touch target height 44px.

**Comment by Camulos** [comment]
In storybook it looks like level 3 button ("quiet" / gjennomsiktig) does not take up the same height/width as level 1 or 2 buttons, when different level buttons are used together this looks and feel like an error. All buttons in same size category should have the same active (touch target) area, even if there is no border og background that visually show this for the level 3 button (until you hover og focus the button).

**Comment by mimarz** [comment]
We'll set the `Button` to have a minimum height in according to its provided size (small, medium or large) and the touch/clickbound to always be 44px in height and width.

**Comment by mimarz** [comment]
Only the "small" `Button` is to small for 44px. 
"Medium" size is 45px (because of fluid padding). 

The default size for `Button` is set to be "small", but maybe this should be "medium"?
